armi/physics/fuelCycle/utils.py

```diff
@@ -0,0 +1,142 @@
+# Copyright 2024 TerraPower, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Geometric agnostic routines that are useful for fuel cycle analysis."""
```

Comment:
"Geometric agnostic routines"... Sort of.

All of these routines assume you have a pin-type reactor. This sentence could be cleaned up.

I'm also wondering if this file name should be changed to `assemblyUtils.py` or `pinUtils.py` or something. Hmm. Just thinking aloud.

Reply:
> All of these routines assume you have a pin-type reactor. This sentence could be cleaned up

Fair. Will revise.

> I'm also wondering if this file name should be changed to assemblyUtils.py or pinUtils.py or something. Hmm. Just thinking aloud

Open to suggestions. We have `hexAssemblyFuelMgmtUtils.py` already and these could just live in there even though they're not hex specific, but still pin-y